### PR TITLE
tests/hassh Add test cases for hassh

### PR DESCRIPTION
**Pull request by MalakhatkoVadym**
This commit adds test cases for the ssh.hassh, ssh.hassh.string, ssh.hassh.server, ssh.hassh.server.string keywords.

### CODE REVIEW

File: tests/ssh-hassh/test.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  min-version: 5.0.0
+features:
+    - RUST
+
+args:
+ - -k none
+
+checks:
+  # Check that we have the following events in eve.json
+  # 2 - alert events
```
Comment (catenacyber) [inline tests/ssh-hassh/test.yaml:11]: The comment says 2 and the code says 4...

File: tests/ssh-hassh/test.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  # 1 - tls event
+  - filter:
+      count: 4
```
Comment (catenacyber) [inline tests/ssh-hassh/test.yaml:14]: Should we check that there is one alert per signature (ie per sid) ?

File: tests/ssh-hassh/test.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      match:
+        event_type: alert
+  - filter:
+      count: 1
+      match:
+        event_type: ssh
```
Comment (catenacyber) [inline tests/ssh-hassh/test.yaml:20]: Should we add a deeper filter to check the value of ssh.hassh ?

File: tests/ssh-hassh/test.yaml (hunk 2 above)
Comment (MalakhatkoVadym) [reply tests/ssh-hassh/test.yaml:14]: Yes, will add now.

File: tests/ssh-hassh/test.yaml (hunk 3 above)
Comment (MalakhatkoVadym) [reply tests/ssh-hassh/test.yaml:20]: it's checked already in rules (rule will match only for specific hassh values).

File: tests/ssh-hassh/test.yaml (hunk 1 above)
Comment (MalakhatkoVadym) [reply tests/ssh-hassh/test.yaml:11]: Sorry, my bad.

File: tests/ssh-hassh/test.yaml (hunk 3 above)
Comment (catenacyber) [reply tests/ssh-hassh/test.yaml:20]: Ok, so should we remove the whole filter of one `event_type == ssh` ?
Or is there more to it ?

Comment (MalakhatkoVadym) [reply tests/ssh-hassh/test.yaml:20]: yes, I think, it can be removed.

**Comment by MalakhatkoVadym** [comment]
https://github.com/OISF/suricata-verify/pull/248

**Pull request closed without merging**